lib/optimizely.rb

```diff
@@ -282,6 +282,28 @@ def is_feature_enabled(feature_flag_key, user_id, attributes = nil)
       false
     end
 
+    def get_enabled_features(user_id, attributes = nil)
+      # Returns the list of features that are enabled for the user.
+      # Args:
+      #   user_id: ID for user.
+      #   attributes: Dict representing user attributes.
+      # Returns:
+      #   A list of the keys of the features that are enabled for the user.
```

Comment:
Just a suggestion: List of feature flag keys

Reply:
Done